src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1393,8 +1381,7 @@ public Territory getRocketAttack(final Collection<Territory> candidates, final T
   }
 
   public static int save(final String filename, final GameData m_data) {
-    try (FileOutputStream fos = new FileOutputStream(filename);
-        ObjectOutputStream oos = new ObjectOutputStream(fos);) {
+    try (FileOutputStream fos = new FileOutputStream(filename); ObjectOutputStream oos = new ObjectOutputStream(fos);) {
```

Comment:
Java7 has trouble closing these in the right order. Please write this as two different try-with-resources blocks (eg: two nest try-autoclose blocks)